### ISSUE TITLE
media datatypes not loaded correctly

### ISSUE THREAD

**Issue by mvdbeek** [galaxyproject/galaxy#10264]
From https://github.com/mvdbeek/tools-iuc/runs/1150658261?check_suite_focus=true:
```
2020-09-22 16:58:28,705 ERROR [galaxy.datatypes.registry] Error importing datatype module galaxy.datatypes.media
Traceback (most recent call last):
  File "/tmp/tmpb9xmi534/galaxy-dev/lib/galaxy/datatypes/registry.py", line 249, in load_datatypes
    datatype_class = getattr(module, datatype_class_name)
AttributeError: module 'galaxy.datatypes.media' has no attribute 'Mov'
2020-09-22 16:58:28,705 ERROR [galaxy.datatypes.registry] Error importing datatype module galaxy.datatypes.media
Traceback (most recent call last):
  File "/tmp/tmpb9xmi534/galaxy-dev/lib/galaxy/datatypes/registry.py", line 249, in load_datatypes
    datatype_class = getattr(module, datatype_class_name)
AttributeError: module 'galaxy.datatypes.media' has no attribute 'Ogv'
2020-09-22 16:58:28,705 ERROR [galaxy.datatypes.registry] Error importing datatype module galaxy.datatypes.media
Traceback (most recent call last):
  File "/tmp/tmpb9xmi534/galaxy-dev/lib/galaxy/datatypes/registry.py", line 249, in load_datatypes
    datatype_class = getattr(module, datatype_class_name)
AttributeError: module 'galaxy.datatypes.media' has no attribute 'Oga'
2020-09-22 16:58:28,715 ERROR [galaxy.datatypes.registry] Error calling method Mov from class <module 'galaxy.datatypes.media' from '/tmp/tmpb9xmi534/galaxy-dev/lib/galaxy/datatypes/media.py'>
Traceback (most recent call last):
  File "/tmp/tmpb9xmi534/galaxy-dev/lib/galaxy/datatypes/registry.py", line 501, in load_datatype_sniffers
    aclass = getattr(module, datatype_class_name)()
AttributeError: module 'galaxy.datatypes.media' has no attribute 'Mov'
```

**Comment by dannon** [comment]
Cleanup from #4949

**Comment by ic4f** [comment]
Fixed in #10269
